cmd/utils/flags.go

```diff
@@ -742,6 +742,11 @@ var (
 		Usage: "Set the value transfer recovery interval (seconds)",
 		Value: 60,
 	}
+	DefaultTxGasLimitFlag = cli.Uint64Flag{
```

Comment:
Now that the user sets the gaslimit with configuration, The `default` prefix may be removed.

Reply:
Thanks for suggestion. Yes now we can set the gaslimit, but that value is **used by default.** So I didn't change the flag name.